### PR TITLE
Added Checksum to S7Blocks 

### PR DESCRIPTION
**Pull request by StefanHasensperling**
Added Checksum to S7Blocks and made Block Atributes to Public Properties, so they become visible in Propertygrids

### CODE REVIEW

File: LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/MC7Converter.cs [hunk 1]
```diff
@@ -189,7 +191,7 @@ internal static S7Block GetAWLBlockBasicInfoFromBlockHeader(byte[] MC7Code, int
                  * xx      = Nertworks
                  */
 
-                retBlock.BlockVersion = new Version(MC7Code[2] / 10, MC7Code[2] % 10);
+                //retBlock.BlockVersion = new Version(MC7Code[2] / 10, MC7Code[2] % 10);
```
Comment (jogibear9988) [inline LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/MC7Converter.cs]: Why have you removed this?

Comment (StefanHasensperling) [inline LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/MC7Converter.cs]: I think i got confused by searching for my problem with the Version field of Online blocks. I confused Version with BlockVersion. I think this line should be reverted to be the same as before.

Anyway What is stored on those Bytes?

Comment (jogibear9988) [inline LibNoDaveConnectionLibrary/PLCs/S7_xxx/MC7/MC7Converter.cs]: Can you create a pull req? I think it's the Version of the Block you see in simatic manager

**Comment by StefanHasensperling** [comment]
No its not. That's where I got confused. The property called "version" is
the one visible in simatic. I don't know what information is stored in
mc7(2).
At least in my cases this byte was always 0x01.

**Comment by StefanHasensperling** [comment]
Actually the Block version is stored in the Blocks footer, not in the header. Its stored after all the MC7 code.

**Comment by jogibear9988** [comment]
Don't know from where I've this Information...

**Pull request merged**